src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1163,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
+     * @param stringWithPrefixSign  Parameter as a string
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefixSign(String stringWithPrefixSign, String sign) {
+        return stringWithPrefixSign.replace(sign, "");
```

Comment:
According to the learning activity you shouldn't do this. Please check [LO-MethodAbstraction](https://github.com/nus-cs2103-AY1617S2/addressbook-level1#abstract-methods-well-lo-methodabstraction)..